docs/windows.md

```diff
@@ -1,5 +1,29 @@
 # Deploying Antrea on Windows
 
+## Table of Contents
+
+<!-- toc -->
+- [Overview](#overview)
+  - [Components that run on Windows](#components-that-run-on-windows)
+  - [Antrea Windows demo](#antrea-windows-demo)
+- [Deploying Antrea on Windows Worker Node](#deploying-antrea-on-windows-worker-node)
+  - [Prerequisites](#prerequisites)
+  - [Installation as a Service (Containerd based runtimes)](#installation-as-a-service-containerd-based-runtimes)
```

Comment:
when I read this section, it still mentioned running kube-proxy in userspace mode as a windows service (nssm). Should this section be updated as well?

Reply:
Updated.